lib/link.ts

```diff
@@ -57,6 +57,8 @@ export const handleLink = async (url: string) => {
     console.error("no url to handle");
     return;
   }
+  // to test opening a transaction:
+  //url = "alby://payment_notification?app_pubkey=ffa0c4a95734243b466d406a1a3ebf4a055bd3a1ef31794e0e3705ee3e6c887b&transaction=%7B%22type%22%3A%22incoming%22%2C%22state%22%3A%22settled%22%2C%22invoice%22%3A%22lnbc200n1p5gphfudqqnp4q228a2ztxkwzaypvzrsh823qngmv97f2v9puwvtsadetypmutyscwpp55s7kzj07uefua5hygqmu34jau0jjp5ck4efvalakj8ztqxntdpkqsp5sxh7kfylhxw6lk336499dpm3w84f6c89h3w8mn95ujm2wnsl4srs9qyysgqcqpcxqyz5vq392s8kt5nhsyywxx0jtjxq3nl6m9yg4zsjqc8mnnj25wmv94hasq8tsl52cnnm02klzy249d4k57qt9pc2ujr9nhf5xq97ne0zgv02gprywph3%22%2C%22description%22%3A%22%22%2C%22description_hash%22%3A%22%22%2C%22preimage%22%3A%22f2fbb43593dd7d7e694d7e330645916c321c730055038fe48f3d6ea858aa9a14%22%2C%22payment_hash%22%3A%22a43d6149fee653ced2e44037c8d65de3e520d316ae52ceffb691c4b01a6b686c%22%2C%22amount%22%3A20000%2C%22fees_paid%22%3A0%2C%22created_at%22%3A1753275708%2C%22expires_at%22%3A1753362108%2C%22settled_at%22%3A1753275741%2C%22settle_deadline%22%3Anull%2C%22metadata%22%3Anull%7D";
```

Comment:
We could use an env variable for testing opening links, and remove the exp:// code. What do you think?